community/kernel/src/docs/ops/io-examples.asciidoc

```diff
@@ -28,9 +28,11 @@ With that setting, you specify the number of bytes available to the page cache,
 The default page memory setting is 50% of the machines memory, after subtracting the memory that is reserved for the Java heap.
 
 For optimal performance, you will want to have as much of your data fit in the page cache as possible.
-You can sum up the size of all the `*store.db*` files in your store file directory, to figure out how big a page cache you need to fit all your data.
-For instance, on a posix system you can look at the total of running `$ du -hc *store.db*` in your `data/graph.db` directory.
-Obviously the store files will grow as you add more nodes, relationships and properties, so configuring more page cache memory than you have data, is recommended when possible.
+You can sum up the size of all the `*store.db*` files in your store file directory, to figure out how big a page cache
```

Comment:
Please keep one sentence per line w/o manual wraps, as elsewhere in the manual to enable easier diffing. Thanks :)